ui/src/tempVars/components/TagKeysTemplateBuilder.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {PureComponent} from 'react'
+
+import KeysTemplateBuilder from 'src/tempVars/components/KeysTemplateBuilder'
+import {TemplateBuilderProps, TemplateValueType} from 'src/types'
```

Comment:
could you put types last as a separate section of imports and also keep components as a separate section (i.e. line-separated)?